mars/dataframe/base/transpose.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+from ..operands import DataFrameOperand, DataFrameOperandMixin
+from ...core import OutputType
+from ..utils import parse_index
+from ...core.operand import OperandStage
+
+
+def reverse(x):
+    if x is None:
```

Comment:
Seems this line is not covered, if `x` won't be None, this function could be removed.